src/CSS/index.css

```diff
@@ -1,3 +1,28 @@
 @tailwind base;
 @tailwind components;
-@tailwind utilities;
\ No newline at end of file
+@tailwind utilities;
+
+/* root */
+
+html {
+  @apply bg-secondary;
+}
+
+/* Scrollbar design */
+
+.scrollbar::-webkit-scrollbar {
+  @apply h-[5px];
+}
+
+.scrollbar::-webkit-scrollbar-track {
+  box-shadow: inset 0 0 6px rgba(0, 0, 0, 0.3);
+}
+
+.scrollbar::-webkit-scrollbar-thumb {
+  background-color: rgb(166 110 252 / var(--tw-bg-opacity));
+  outline: 1px solid slategrey;
+}
+
+.scrollbar::-webkit-scrollbar-thumb:hover {
+  background: #555;
+}
```

Comment:
Could add it to Tailwind css?

Reply:
All CSS

Reply:
The whole scrollbar design section ir just this part 
 coz as per ik these arnt availaible for Tailwind 
 whenever i work with tailwind i use these only

Reply:
> The whole scrollbar design section or just this part coz as per ik these arnt availaible for Tailwind whenever i work with tailwind i use these only

Yeah! I checked. Currently, you don't need to do anything about it. It's working properly.
Thanks for your awesome support 
![image](https://user-images.githubusercontent.com/88102392/223125965-aa11373c-3780-41bd-becb-b0994e95711f.png)